### DISCUSSION TITLE
Help wanted setting up LaTeX compiling protocol

### DISCUSSION THREAD

**Discussion by mariogarcc** [James-Yu/LaTeX-Workshop#3138] [Q&A]
I apologize if this is not really a issue, but more of a call for help because I'm really lost in how to use LaTeX command tools and all that sort of stuff and I don't really have time to investigate it deeply on my own.

I am trying to have a setup like the following:

    subject/ # parent folder
      |-- assets/ # images stored here
      |-- books/
      |   |-- bookone/
      |   |   |-- bookone.bib
      |   |-- booktwo/
      |   |   |-- booktwo.bib
      |-- source/
      |   |-- dist/ # here should go all the auxiliary files including synctex and fls
      |   |-- partone/
      |   |   |-- partone.tex
      |   |-- parttwo/
      |   |   |-- parttwo.tex
      |-- partone.pdf
      |-- parttwo.pdf

I tried messing with `"latex-workshop.latex.outDir"` and `"latex-workshop.latex.tools"` settings but apparently I also have to use recipes for command lines and I really have no idea on how to use them. I'm also getting errors on finding `.bib` files when I change `outDir`, etc.

Please help.

**Comment by jlelong** [comment]
Separating the auxiliary files from the generated `pdf` makes the configuration quite complicated. I suggest you should try to set up your workflow outside of the extension and then add the correct options in `"latex-workshop.latex.tools"`. If you need help to set up your workflow, ask a LaTeX forum.
